backend/src/cliInterface.ts

```diff
@@ -22,6 +22,15 @@ class CLIInterface extends CLI {
           Tracker.event('device', 'delete', this.data.device.name)
         }
         break
+
+      case 'registration':
```

Comment:
Should these "magic" keys instead be constants/globals stored in a central location?

Reply:
Yea, they are type-checked so they feel pretty safe. Could add as globals tool.